### DISCUSSION TITLE
Streets/Details not loading in the 'Places' map; Broken map in local Photoprism

### DISCUSSION THREAD

**Discussion by amirootyet** [photoprism/photoprism#3062] [Q&A] [answered]
### Preface
My issue is the same as the one discussed [here](https://github.com/photoprism/photoprism/discussions/2096).

However, the cause and fix is not the same (as far as I've tested.)

### System
- Ubuntu 22.04
- Docker version 20.10.22, build 3a2c30b
- Pihole (DNS) (shouldn't matter since I've tried disabling Pihole to see if map details load but no luck)
- Firefox (disabled Ublock Origin, but browser shouldn't matter since it loads map details in the Photoprism online demo just fine)

### Issue

On a fresh installation of Photoprism operating out of a docker container based on a `docker-compose.yml`, the Places tab in my Photoprism is missing all the street details as shown below:

![Screenshot from 2023-01-04 18-20-00](https://user-images.githubusercontent.com/46239148/210674243-b510d67c-31a0-4c71-b8ff-8b8c38ec5d29.png)

![Screenshot from 2023-01-04 18-20-58](https://user-images.githubusercontent.com/46239148/210674318-d12099e9-577d-41b3-8d4d-7e15d3d9197f.png)

It used to work ~3 months or so ago and then suddenly stopped working one day. I run a `watchtower` service that updates containers to latest version. Perhaps, after an update something became an issue.

### What I've tried so far
1. As suggested in issue [here](https://github.com/photoprism/photoprism/discussions/2096), I've tried restarting my browser
2. My browser blocking WebGL does not seem to be the problem since:
    - I've tried different non-privacy-oriented browsers like firefox with no adblock and default browser on my phone
    - I've looked at browser web console logs, and there's no content block related errors

![Screenshot from 2023-01-04 18-29-29](https://user-images.githubusercontent.com/46239148/210675169-8e1705b1-fffc-431b-9ca8-e6f434e30558.png)

3. **The [Photoprism demo](https://demo.photoprism.app/library/places) works fine for me. I've noticed that it fetches details from `https://api.maptiler.com` in Network tab on my browser console:**

![Screenshot from 2023-01-04 18-34-53](https://user-images.githubusercontent.com/46239148/210675687-d2c5e178-a246-413a-a8ac-b6db1e1f27bd.png)

**By contrast, my local instance does not even try to load from this API as shown below.**

![Screenshot from 2023-01-04 18-36-08](https://user-images.githubusercontent.com/46239148/210675753-0f5f50f3-8df8-4a12-96b2-ae2183735fee.png)

4. Enabled debug logs in my docker compose to see any related errors in the container and I don't see anything useful.

![Screenshot from 2023-01-04 18-38-56](https://user-images.githubusercontent.com/46239148/210676037-e50ef0ce-937f-4977-b3ae-02081b280335.png)

**Comment by amirootyet** [comment, marked as answer]
UPDATE: Just found [this discussion](https://github.com/photoprism/photoprism/issues/2998) and [this one](https://www.reddit.com/r/selfhosted/comments/zf0e4b/selfhosted_photo_app_with_map_view_photoprism/). So are the city/street details in Places now a paid feature?

**Reply by lastzero** [reply]
The high-quality maps have been a paid feature for a long time, but were still available for testing. Since many users didn't realize this and simply used the commercial maps for free, this was recently changed. Please see the feature comparison on our website for reference: https://photoprism.app/editions#compare

As this allows us to make more features available to the public, we encourage all users to support our mission by signing up as a sponsor or contributing to the open source project. Ultimately, that's what's best for the product and the community. Thank you very much! ❤️

See also:
- #2998

**Reply by lastzero** [reply]
I've added this question to the FAQ in the User Guide, in addition to the [Getting Started FAQ](https://docs.photoprism.app/getting-started/faq/#are-the-keys-for-using-interactive-world-maps-provided-free-of-charge) where it was already covered:

https://docs.photoprism.app/user-guide/faq/
